update_factorio.py

```diff
@@ -138,11 +138,28 @@ def verbose_aware_exec(exec_args, verbose=False):
         raise
 
 
+def get_username_token(factoriopath):
+    with open(os.path.normpath(factoriopath + '/../../../player-data.json')) as data_file:  
```

Comment:
The headless server writes user/token to player-data.json, atleast on 0.15.x. It's copied from server-settings.json.